documentation/spring/guide-service-client-spring.asciidoc

```diff
@@ -0,0 +1,279 @@
+:toc: macro
+:icons: font
+toc::[]
+
+= Service Client
+
+This guide is about consuming (calling) services from other applications (micro-services). For providing services, see the link:guide-service-layer.asciidoc[Service-Layer Guide]. Services can be consumed in the link:guide-client-layer.asciidoc[client] or the server. As the client is typically not written in Java, you should consult the according guide for your client technology. In case you want to call a service within your Java code, this guide is the right place to get help.
```

Comment:
Since this file is now in a different folder, the internal links pointing to other guides will not work. Therefore, please change the path in the link. Also check the other links.